test/regression/base/generic/trivial_unsat_core2_full.smt2.expected.out

```diff
@@ -0,0 +1,2 @@
+unsat
+( b1 a3 )
```

Comment:
This is not correct. We should not mix terms and name together.
`cvc5` returns 
```
unsat
(
b1
(not b1)
)
```

Reply:
Sounds reasonable.